Calculator/Calculator/Model/LinkedList.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LinkedList.swift
+//  Calculator
+//
+//  Created by Daehoon Lee on 2023/05/31.
+//
+
+struct LinkedList<T> {
+    var head: Node<T>?
+    var tail: Node<T>?
+    
+    init(head: Node<T>?) {
+        self.head = head
+    }
+    
+    mutating func append(_ data: T) {
+        guard head != nil
+        else {
```

Comment:
```suggestion
        guard head != nil else {
```

보통 `else`의 앞에서 개행을 하지는 않는 것 같습니다. 또한 `removeFirst()` 메서드의 `guard` 문은 개행이 되어있지 않아서 통일이 필요한 부분일 것 같습니다.

Reply:
`removeFirst` 메서드에서 달랐던 이유는 여기에선 `else`문 내부에 동작이 `return`만 있었기 때문이었습니다. `else`문에 여러 동작이 필요한 경우에 위와 같은 convention을 사용했습니다. 건디가 가이드라인으로 보시는 convention 문서가 있을까요?

Reply:
[Statements](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/statements) 등의 문서가 기본 참고자료가 될 것 같습니다. 하지만 일관적인 컨벤션을 사용한다면 이 부분은 사바사 케바케가 될 것 같습니다.

Reply:
말씀해 주신 내용처럼 공식 문서 위주의 convention을 사용 해보도록 하겠습니다.😊